### DISCUSSION TITLE
BeagleBone® Introduction | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#389] [Q&A]
# BeagleBone® Introduction | Seeed Studio Wiki

BeagleBone® Introduction

https://wiki.seeedstudio.com/XIAO_BLE/

**Comment by HarrisBirch** [comment]
Hi, I was having an issue with the I2C device with Circuitython. I am using 2 devices (rv3028 RTC and MCP23008 IO expander) and they both have different addresses, but when I run my code there is an error saying "A5 in use." A5 is the SCL pin. I know you should be able to run 2 devices with different addresses, and I would be grateful for any help.
Thanks!

**Reply by TravisSeeed** [reply]
I just share some opinions with you. It sounds like there may be a conflict between the I2C addresses assigned to your RTC and IO expander devices, which are causing the SCL pin to be in use. I suggest you to check the I2C addresses of your devices: Make sure that the RTC and IO expander are configured with different I2C addresses as expected. You can refer to the datasheets of the RTC and IO expander to ensure the correct addresses are being set. Also, double-check your wiring to ensure that the devices are connected to the correct pins.
